frontend/public/components/monitoring.tsx

```diff
@@ -1099,28 +1081,20 @@ const MonitoringListPage = connect(filtersToProps)(
           <Helmet>
             <title>Alerting</title>
           </Helmet>
-          <div className="co-m-pane__filter-bar">
+          <div className="co-m-pane__body">
             {CreateButton && (
-              <div className="co-m-pane__filter-bar-group">
+              <div className="co-m-pane__createLink co-m-pane__createLink--no-title">
```

Comment:
```suggestion
              <div className="co-m-pane__createLink--no-title">
```
We have completely eliminated that CSS rule.